go/debug-multipart.sh

```
@@ -0,0 +1,39 @@
+    # config.jsonを一時的に修正（バックアップを作成）
+    cp config.json config.json.bak
+    
+    # jqを使ってリージョンを更新
+    jq --arg region "$REGION" '.aws.region = $region' config.json > config.json.tmp && mv config.json.tmp config.json
+    
+    # テストを実行
+    echo "Running multipart upload test..."
+    go run cmd/multipart-test/main.go \
+        -source ../test-data/test-150mb.bin \
+        -bucket "$BUCKET" \
+        -key test/large-$(date +%Y%m%d-%H%M%S).bin
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)